packages/dds/tree/src/test/simple-tree/prepareForInsertion.spec.ts

```diff
@@ -59,4 +84,253 @@ describe("prepareForInsertion", () => {
 		assert.equal(deep1, root[0]);
 		assert.equal(deep2, root[1]);
 	});
+
+	describe("Stored schema validation", () => {
```

Comment:
Moved from src/test/simple-tree/toMapTree.spec.ts and refactored to target prepareForInsertionContextless instead of toMapTree